.gitmodules

```diff
@@ -13,3 +13,10 @@
 [submodule "packages/contracts-bedrock/lib/forge-std"]
 	path = packages/contracts-bedrock/lib/forge-std
 	url = https://github.com/foundry-rs/forge-std
+[submodule "packages/contracts-bedrock/lib/safe-contracts"]
+	path = packages/contracts-bedrock/lib/safe-contracts
+	url = https://github.com/safe-global/safe-contracts
+  branch = v1.4.0
```

Comment:
looks like this was manually edited because of tabs vs spaces

Reply:
fixed.